polyfills.ts

```
@@ -19,3 +20,26 @@ if (!global.crypto.subtle) {
+
+function randomUUID() {
+  if (NativeModules.RandomUuid) {
+    return NativeModules.RandomUuid.getRandomUuid();
+  }
+
+  // Expo SDK 48+
+  if (
+    global.expo &&
+    global.expo.modules &&
+    global.expo.modules.ExpoCrypto &&
+    global.expo.modules.ExpoCrypto.randomUUID
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)